sdk.yaml

```diff
@@ -374,6 +374,11 @@ REST:
       .specification: RSA8e
   Channel:
     .specification: [RSN1, RTC3]
+    Encryption:
+      .specification: [RSL7, TB2b, RSN3c]
+      .synopsis: |
+        The option to configure a `cipher`, specifying encryption algorithm and attributes.
```

Comment:
Is it appropriate to documented exactly what kind of encryption we're talking about here (i.e. AES-CBC with either 128 or 256 bit keys)?

Reply:
I think that makes sense. I'll borrow words from here: https://ably.com/docs/api/realtime-sdk/encryption#cipher-params

Reply:
https://github.com/ably/features/pull/29/commits/29353be618e2905f77926d7927e5f176d1a91244

Reply:
Resolving this conversation on @lmars' behalf so that I can get this pull request landed.